### ISSUE TITLE
[Request] Feature (or Tutorial on how) to connect to custom rich presences

### ISSUE THREAD

**Issue by dreamnav** [NaamloosDT/foo_discord#18]
See title. I figured I should request this so that people can add their own images and change things around without nagging you to add it to the component itself

**Comment by Naamloos** [comment]
I'll leave this one open for now. Will come back when I get time

**Comment by ensorcell** [comment]
Could this be done by letting user fill in their own application ID, and having the image options be something universal like "1", "2", "3", etc?

**Comment by dreamnav** [comment]
Yeah, something like that's along the lines of what I was thinking

**Comment by Naamloos** [comment]
I personally wouldn't prefer it like that. I'll get back after work. let's hope I correctly installed the windows SDKs

**Comment by Naamloos** [comment]
I'm a living paradox, I'm actually planning on allowing something like this sometime soon..
